components/typography/Heading1.module.css

```diff
@@ -5,6 +5,36 @@
   line-height: 19px;
 }
 
+.detailsPage {
+  font-size: 28px !important;
```

Comment:
It's a bad idea to use `!important` this makes styling more diffult for the next person. Did you try finding another way? 

Perhaps you need to change the order of the CSS so the right values apply. If you are having trouble debugging this we can take a look together